src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -398,6 +406,9 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
         if(getShmSize() != null) { // 0 is valid input
             containerCommand.hostConfig.withShmSize(getShmSize())
         }
+        if (getAutoRemove() != null) { // explicit false is valid
```

Comment:
You're probably just following convention but you don't need to do the explicit null check as groovy will handle things. A simple `if (getAutoRemove()) { ... }` should suffice.

Reply:
I was avoiding groovy truthiness in case someone explicitly sets false and the default behavior of the upstream libraries change (not that that seems likely)

Reply:
Ok fair enough.